neurobooth_os/tasks/fixations.py

```diff
@@ -91,6 +92,7 @@ def present_task(
             self.sendMessage(self.marker_trial_end, False)
 
         self.sendMessage(self.marker_task_end)
+        self.sendCommand('clear_screen 0')
```

Comment:
Since we have this "magic string" (a la unnamed "magic numbers") invocation pop up in a few places, I would make this into it's own brief method `clear_tablet_screen()` for abstraction/clarity.

Reply:
Done in the Eyelink_HostPC class